pandas/io/tests/parser/common.py

```diff
@@ -1236,3 +1233,17 @@ def test_iteration_open_handle(self):
                     result = self.read_table(f, squeeze=True, header=None)
                     expected = Series(['DDD', 'EEE', 'FFF', 'GGG'], name=0)
                     tm.assert_series_equal(result, expected)
+
+    def test_read_duplicate_names(self):
+        # See gh-7160
+        data = "a,b,a\n0,1,2\n3,4,5"
+        df = self.read_csv(StringIO(data))
```

Comment:
where is test when ``manlge_dupe_cols=False`` and you have dupe ``names``, it raises ``ValueError`` right?